org.gnome.Podcasts.json

```diff
@@ -8,17 +8,11 @@
     ],
     "command" : "gnome-podcasts",
     "finish-args" : [
-        "--filesystem=xdg-run/dconf",
```

Comment:
Add the migration metadata. `"--metadata=X-DConf=migrate-path=/org/gnome/whatever/",`

Reply:
Good catch, thanks!